clients/wallet/src/operations.rs

```diff
@@ -197,16 +199,8 @@ pub mod redeem_request_tests {
 
 	const IS_PUBLIC_NETWORK: bool = false;
 
-	const DEFAULT_SOURCE_PUBLIC_KEY: &str =
```

Comment:
@ebma I removed the hard-coded accounts

Reply:
And it works with the other one? That's nice

Reply:
Yes it works. All tests in this file will use the keys in the env.